network/src/operator.rs

```diff
@@ -204,42 +161,40 @@ impl<N: Network, E: Environment> Operator<N, E> {
                 error!("Missing operator address. Please specify an Aleo address in order to operate a pool");
             }
         }
-
-        Ok(operator)
     }
 
     /// Returns an instance of the operator router.
-    pub fn router(&self) -> OperatorRouter<N> {
-        self.operator_router.clone()
+    pub fn router(&self) -> &OperatorRouter<N> {
+        &self.operator_router
     }
 
     /// Returns all the shares in storage.
     pub fn to_shares(&self) -> Vec<((u32, Record<N>), HashMap<Address<N>, u64>)> {
-        self.state.to_shares()
+        self.operator_state.to_shares()
     }
 
     /// Returns the shares for a specific block, given the block height and coinbase record commitment.
     pub fn get_shares_for_block(&self, block_height: u32, coinbase_record: Record<N>) -> Result<HashMap<Address<N>, u64>> {
-        self.state.get_shares_for_block(block_height, coinbase_record)
+        self.operator_state.get_shares_for_block(block_height, coinbase_record)
     }
 
     /// Returns the shares for a specific prover, given the prover address.
     pub fn get_shares_for_prover(&self, prover: &Address<N>) -> u64 {
-        self.state.get_shares_for_prover(prover)
+        self.operator_state.get_shares_for_prover(prover)
     }
 
     ///
     /// Returns a list of all provers which have submitted shares to this operator.
     ///
     pub fn get_provers(&self) -> Vec<Address<N>> {
-        self.state.get_provers()
+        self.operator_state.get_provers()
     }
 
     ///
     /// Performs the given `request` to the operator.
     /// All requests must go through this `update`, so that a unified view is preserved.
     ///
-    pub(super) async fn update(&self, request: OperatorRequest<N>) {
+    pub async fn update(&self, request: OperatorRequest<N>) {
```

Comment:
Revert: why is `super` removed? The goal is not ensure this is not callable outside of this scope, out of order executions lead to node halts